src/task/SubscribeAndReceiveSubscribeAckTask.ts

```diff
@@ -88,14 +109,36 @@ export default class SubscribeAndReceiveSubscribeAckTask extends BaseTask {
       isSendingStreams,
       this.context.videoStreamIndex.localStreamDescriptions(),
       // TODO: handle check-in mode, or remove this param
-      true
+      true,
+      compressedSDPOffer
     );
     this.context.logger.info(`sending subscribe: ${JSON.stringify(subscribe)}`);
     this.context.signalingClient.subscribe(subscribe);
 
     const subscribeAckFrame = await this.receiveSubscribeAck();
     this.context.logger.info(`got subscribe ack: ${JSON.stringify(subscribeAckFrame)}`);
+
+    let decompressedText = '';
+    if (subscribeAckFrame.compressedSdpAnswer && subscribeAckFrame.compressedSdpAnswer.length > 0) {
```

Comment:
nit: subscribeAckFrame.compressedSdpAnswer?.length > 0

Reply:
I tried to do that, but I was not able to get the branch coverage to 100% with that. so reverting to the explicit notation.